test/fuzz/test_runner.py

```diff
@@ -152,6 +165,40 @@ def main():
         )
 
 
+def generate_corpus_seeds(build_dir, seed_dir, targets, n):
+    """Generates new corpus seeds.
+
+    Run {n} {targets} at a time without input, and outputs the generated corpus
+    seeds to {seed_dir}.
+    """
+    logging.info(f"Generating corpus seeds to {seed_dir}")
+
+    with ThreadPoolExecutor(max_workers=n) as fuzz_pool:
+        def job(command):
+            try:
+                logging.debug("Running {}..".format(" ".join(command)))
+                logging.debug("Output: {}\n\n".format(
+                    subprocess.run(command, check=True,
+                                   stderr=subprocess.PIPE, timeout=n*5).stderr
```

Comment:
Any reason to set a time based timeout here, but max runs by number below?

Reply:
Yeah, the max runs is for the number of targets per batch to run in parallel, while this timeout is because some targets will have a harder time finding new coverage before some others and is to avoid obstructing the queue with them.

Reply:
Hmm actually i used something different and timed out slow fuzzing processes because they were blocking the whole next batch.. With a thread pool maybe it's not necessary ?